iocore/net/YamlSNIConfig.h

```diff
@@ -26,25 +26,30 @@
 
 #include "tsconfig/Errata.h"
 
-constexpr char TS_fqdn[]                 = "fqdn";
-constexpr char TS_disable_H2[]           = "disable_h2";
-constexpr char TS_verify_client[]        = "verify_client";
-constexpr char TS_tunnel_route[]         = "tunnel_route";
-constexpr char TS_verify_origin_server[] = "verify_origin_server";
-constexpr char TS_client_cert[]          = "client_cert";
-constexpr char TS_ip_allow[]             = "ip_allow";
+constexpr char TS_fqdn[]                     = "fqdn";
```

Comment:
```
#define X(ID) constexpr char TS_##ID #ID; 
X(fqdn)
```

Reply:
?

Reply:
Ok, I see.  A means to avoid double typing (and messing up) the same string.

Reply:
It's just a thought, I suppose some might find it overly tricky.  To me, this sort of technique makes it more clear when there's a consistent pattern.  Also helps avoid typos.